test/test_image_plot.py

```diff
@@ -1,16 +1,19 @@
 # -*- coding: utf-8 -*-
 #
 import helpers
+from matplotlib import rcParams
+from matplotlib import pyplot as pp
+import pytest
+import os
 
+try:
+    from PIL import Image
+except ImportError:
+    raise RuntimeError('PIL must be installed to run this example')
 
-def plot():
-    from matplotlib import rcParams
-    from matplotlib import pyplot as pp
-    import os
-    try:
-        from PIL import Image
-    except ImportError:
-        raise RuntimeError('PIL must be installed to run this example')
+#the picture 'lena.png' with origin='lower' is flipped upside-down. So it has to be upside-down in the pdf-file as well.
+
+def plot1(): #test for monochrome picture
```

Comment:
two tests for rgb and monochrome picture. Now we are able to supervise changes in matplotlib.imsave and PIL.image.save. Hopefully, sometimes the origin parameter will be considered, so we can get rid of the numpy.flipud in matplotlib2tikz.image